website/blog/2024-08-06-langgraph-model-from-code/index.md

```diff
@@ -264,7 +261,7 @@ Before concluding, let's demonstrate [MLflow tracing](https://mlflow.org/docs/la
 
 MLflow Tracing is a feature that enhances LLM observability in your Generative AI (GenAI) applications by capturing detailed information about the execution of your application’s services. Tracing provides a way to record the inputs, outputs, and metadata associated with each intermediate step of a request, enabling you to easily pinpoint the source of bugs and unexpected behaviors.
 
-Start the MLflow server as outlined in the [tracking server docs](https://mlflow.org/docs/latest/tracking/server.html). After entering the MLflow UI, we can see our experiment and corresdponding traces.
```

Comment:
Can we remove `with mlflow.start_run():` call at L211? We don't need a run to create traces. It might give false impression to users that they need a run for tracing/loading model.